core/db/migrate/20151010003252_add_foreign_keys_for_taxons_prototypes.rb

```diff
@@ -0,0 +1,7 @@
+class AddForeignKeysForTaxonsPrototypes < ActiveRecord::Migration
+  def change
+    { spree_taxons: :taxon_id, spree_prototypes: :prototype_id }.each do |to_table, column|
+      add_foreign_key :spree_prototype_taxons, to_table, column: column, on_delete: :cascade
```

Comment:
I like our DB maintaining referential integrity. It does a much better job of it than ActiveRecord. I also feel that cascade is the correct `on_delete` to use here.